AutomationHelpers/test/packages.config

```diff
@@ -3,5 +3,5 @@
   <package id="NSubstitute" version="2.0.3" targetFramework="net45" />
   <package id="NUnit" version="3.8.1" targetFramework="net45" />
   <package id="NUnit.ConsoleRunner" version="3.7.0" targetFramework="net45" />
-  <package id="Ranorex.ReportToPDF" version="1.0.16" targetFramework="net45" />
+  <package id="Ranorex.ReportToPDF" version="1.0.17" targetFramework="net45" />
```

Comment:
According to nuget, there is no 1.0.17 version out, yet:
https://www.nuget.org/packages/Ranorex.ReportToPDF/